content/core-features/authentication.textile

```diff
@@ -21,15 +21,17 @@ redirect_from:
   - /general/_authentication_capabilities
 ---
 
-Both the "REST client library":/rest/authentication and the "Realtime client library":/realtime/authentication use common authentication mechanisms. The two schemes supported by Ably are "Basic Authentication":#basic-authentication, which uses your private "API key":https://knowledge.ably.com/what-is-an-app-api-key, and "Token Authentication":#token-authentication, which uses short-lived tokens for access which are periodically renewed.
+Both the "REST client library":/rest/authentication and the "Realtime client library":/realtime/authentication use common authentication mechanisms. The two schemes supported by Ably are "basic authentication":#basic-authentication, which uses your Ably "API key":https://knowledge.ably.com/what-is-an-app-api-key, and "token authentication":#token-authentication, which uses short-lived tokens for access. These tokens are periodically renewed.
```

Comment:
I wonder if we should take the opportunity to start calling API key based authentication what it is: "key authentication" (or "secret key authentication", or "API key authentication") rather than "basic authentication".

I say this because there is nothing in our SDK API that uses the term "basic", it's just an implementation detail that the API key is often set as the HTTP Basic Authentication header, but it may also be set as the `key` query parameter in the URL rather than using HTTP basic auth.

This is obviously quite a departure from the current nomenclature, but I do think it's an improvement.

Reply:
Thanks for the feedback Lewis. That would indeed be a big change and beyond the scope of this PR/ticket. I will capture in another ticket though and add it to our Backlog.